pkgs/by-name/xa/xarcan/package.nix

```
@@ -85,6 +86,10 @@ stdenv.mkDerivation (finalPackages: {
     xtrans
   ];
 
+  patches = [
+    ./fix-segfault.patch
```

1. Was this patch upstreamed?
2. Comment why this patch is needed
3. Prefix the patch with a number (since patches need to be ordered):

```suggestion
    ./0001-fix-segfault.patch
```

---

1. Not yet
2. Because it segfaults otherwise because a pointer is not checked. I started this as a test if I can make Xarcan work over arcan_sdl
3. I'll do it

---

https://github.com/letoram/xarcan/pull/12